include/cudaq/Optimizer/Transforms/Passes.td

```diff
@@ -305,11 +305,12 @@ def QuakeSynthesize : Pass<"quake-synth", "mlir::ModuleOp"> {
   let constructor = "cudaq::opt::createQuakeSynthesizer()";
 }
 
-def QuakeAddDeallocs : Pass<"quake-add-deallocs", "mlir::func::FuncOp"> {
+def QuakeAddDeallocs : Pass<"add-dealloc", "mlir::func::FuncOp"> {
```

Comment:
If we're changing the pass string name, I'd imagine we'd want to change the pass name (i.e. QuakeAddDeallocs --> AddDealloc) in order to keep things consistent.